### DISCUSSION TITLE
Any news on YOLO8 mask sliced inference?

### DISCUSSION THREAD

**Discussion by EmKarr** [obss/sahi#995] [Q&A] [answered]
Hi there,

I know this topic has been under discussion already, nevertheless I would like to ask if there is some news on the implementation of get_sliced_prediction() with YOLO8 segmentation masks. Is this a feature to expect soon?

Best,
Michael

**Comment by fcakyon** [comment, marked as answer]
Segmentation mask support for yolov8, mmdetection, detectron2, torchvision models is live with the latest release! @EmKarr https://github.com/obss/sahi/discussions/1051

Check the demo notebook on how it works: https://github.com/obss/sahi/blob/main/demo/inference_for_yolov8.ipynb

**Reply by ardithho** [reply]
While it is included in the repo, it doesn't seem to be updated in the the python library package. Also, will there be support for pose models in the future?
